react/new/src/components/MainNavigation/index.tsx

```diff
@@ -1,9 +1,15 @@
 import { Menu, Group, Center, Burger, Container } from '@mantine/core'
 import { useDisclosure } from '@mantine/hooks'
-import { IconChevronDown } from '@tabler/icons-react'
+import {
+  IconChevronDown,
+  IconChevronRight,
+  IconSettings,
+} from '@tabler/icons-react'
 import { Link } from 'react-router-dom'
+import { useState } from 'react'
 import classes from './main-navigation.module.scss'
```

Comment:
What's the difference between `import classes` and `import { useState}` with the curlies?

Reply:
When we use `import { useState } from "react";` we're using a named import, calling the specific export from the library that exports it. `useState` is a core hook from React for, you might have already guessed, state management. It's only useful when you aren't managing a ton of state, but once you start to scale up you need to start thinking about using a `context` or something even more robust like `Redux` 

The `import classes from './main-navigation.module.scss'` imports the default export from the scss file. In this context it pulls all the classes defined in that stylesheet and gives us access to them in the component, so like `classes.mobileMenu` for example, then we can apply that class to an element in the DOM and thus apply the styles associated with the class.